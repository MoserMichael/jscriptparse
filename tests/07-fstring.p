    this=3
    that=4

    # testing f-strings
    print("sum: {this + that} product: {this * that} diff: {this - that}.")
