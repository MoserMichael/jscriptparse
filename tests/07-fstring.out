sum: 7 product: 12 diff: -1.
